### PR TITLE
feat: User addresses

### PR DESCRIPTION
**Pull request by Haider8**

### CODE REVIEW

**Comment by Haider8** [comment]
@iamareebjamal please review

File: app/db_models/address.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from geoalchemy2.types import Geography
+
+
+class Address(Base):
+    """To store the addresses of user.
+    
+    `tag` can be Home, Work, Other etc.
+    """
+
+    __tablename__ = 'addresses'
```
Comment (iamareebjamal) [inline app/db_models/address.py]: Should be singular

Comment (Haider8) [reply app/db_models/address.py]: @iamareebjamal I have followed this same convention in some other tables also. Why is this wrong?

Comment (iamareebjamal) [reply app/db_models/address.py]: https://stackoverflow.com/a/5841297/3309666

Addresses sounds weirder than Address

**Pull request merged**